### PR TITLE
Change Makefile and README to single-binary eventing-controller

### LINKED ISSUE #11343: Update the Makefile and README.md for the new eventing-backend, to reflect the current implementation:
Update the Makefile and README.md for the new eventing-backend, to reflect the current implementation:
```
components/eventing-controller/Makefile
components/eventing-controller/README.md
```

_Originally posted by @radufa in https://github.com/kyma-project/kyma/pull/11256#r637055768_

### PR DESCRIPTION
**Pull request by themue**
**Description**

Changes proposed in this pull request:

- Adopt single-binary eventing controller in Makefile and README.md

**Related issue(s)**

Resolves #11343

### CODE REVIEW

**Review by alexandra-simeonova — DISMISSED** [review]

File: components/eventing-controller/README.md [hunk 1]
```diff
@@ -55,14 +41,13 @@ This section explains how to use the Eventing Controller. It expects the followi
     | WEBHOOK_TOKEN_ENDPOINT | The Kyma public endpoint to provide Access Tokens.                              | BEB     |
     | DOMAIN                 | The Kyma cluster public domain.                                                 | BEB     |
     | NATS_URL               | The URL for the NATS server.                                                    | NATS    |
-    | APP_LOG_FORMAT         | The format of the Application logs.                                             | Both    |
-    | APP_LOG_LEVEL          | The level of the Application logs.                                              | Both    |
```
Comment (marcobebway) [inline components/eventing-controller/README.md:59]: We need those env vars. Please revert the deletion.

File: components/eventing-controller/README.md [hunk 2]
```diff
@@ -55,14 +41,13 @@ This section explains how to use the Eventing Controller. It expects the followi
...
 
 The additional command line arguments are:
 
     | Flag                  | Description                                               | Default Value | Backend |
     | --------------------- | --------------------------------------------------------- | ------------- | ------- |
-    | metrics-addr          | The address the metric endpoint binds to.                 | :8080         | Both    |
+    | metrics-addr          | The address the metric endpoint binds to.                 | :8080         | both    |
+    | enable-debug-logs     | Enable debug logs.                                        | false         | both    |
```
Comment (marcobebway) [inline components/eventing-controller/README.md:50]: `enable-debug-logs` is not needed anymore. Please remove it.

File: components/eventing-controller/README.md (hunk 1 above)
Comment (themue) [reply components/eventing-controller/README.md:59]: Will do. Strange, why had they being removed. 🤔 Possibly git error on my side.

**Comment by radufa** [comment]
The https://github.com/kyma-project/kyma/blob/main/resources/eventing/README.md should be updated too, referring to the new https://github.com/kyma-project/kyma/blob/main/resources/cluster-essentials/files/eventingbackends.eventing.kyma-project.io.crd.yaml

**Review by radufa — COMMENTED** [review]
This doesn't work on my machine: https://github.com/kyma-project/kyma/blob/058718b1cfe932aaa24e1d56a62c820ddf8161cc/components/eventing-controller/README.md#start-the-controller-locally

**Comment by themue** [comment]
@radufa Added it.

**Comment by themue** [comment]
Some infos for finishing the README env vars in local run mode are needed.

File: components/eventing-controller/README.md [hunk 3]
```diff
@@ -55,14 +41,14 @@ This section explains how to use the Eventing Controller. It expects the followi
     | WEBHOOK_TOKEN_ENDPOINT | The Kyma public endpoint to provide Access Tokens.                              | BEB     |
     | DOMAIN                 | The Kyma cluster public domain.                                                 | BEB     |
```
Comment (sayanh) [inline components/eventing-controller/README.md:42]: I think we need to document the env vars mentioned [for backend-controller](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/env/backend_config.go#L13-L27) as well.

File: components/eventing-controller/README.md [hunk 4]
```diff
@@ -14,82 +14,80 @@ This component contains controllers for various CustomResourceDefinitions relate
...
+| `PUBLISHER_PORT_NUM`          | The port number of the Event Publisher Proxy itself.                            |
+| `PUBLISHER_METRICS_PORT_NUM`  | The port number of the Event Publisher Proxy metrics.                           |
+| `PUBLISHER_SERVICE_ACCOUNT`   | The service account of the Event Publisher Proxy.                               |
+| `PUBLISHER_REPLICAS`          | The number of replicas of the Event Publisher Proxy.                            |
+| `PUBLISHER_REQUESTS_CPU`      | The CPU requests of the Event Publisher Proxy.                                  |
+| `PUBLISHER_REQUESTS_MEMORY`   | The memory requests of the Event Publish Proxy.                                 |
+| `PUBLISHER_LIMITS_CPU`        | The CPU limits of the Event Publisher Proxy.                                    |
+| `PUBLISHER_LIMITS_MEMORY`     | The memory limits of the Event Publisher Proxy.                                 |
+| **For NATS**                  |                                                                                 |
+| `NATS_URL`                    | The URL for the NATS server.                                                    |
```
Comment (sayanh) [inline components/eventing-controller/README.md:51]: `EVENT_TYPE_PREFIX`, `MAX_IDLE_CONNS`, `MAX_CONNS_PER_HOST`, `MAX_IDLE_CONNS_PER_HOST`, `IDLE_CONN_TIMEOUT` are missing. You may find them [here](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/env/nats_config.go#L10-L26).

File: components/eventing-controller/README.md [hunk 5]
```diff
@@ -14,82 +14,80 @@ This component contains controllers for various CustomResourceDefinitions relate
...
+| **For BEB**                   |                                                                                 |
+| `TOKEN_ENDPOINT`              | The Authentication Server Endpoint to provide Access Tokens.                    |
+| `WEBHOOK_CLIENT_ID`           | The Client ID used by webhooks to acquire Access Tokens from Kyma.              |
+| `WEBHOOK_CLIENT_SECRET`       | The Client Secret used by webhooks to acquire Access Tokens from Kyma.          |
+| `WEBHOOK_TOKEN_ENDPOINT`      | The Kyma public endpoint to provide Access Tokens.                              |
+| `DOMAIN`                      | The Kyma cluster public domain.                                                 |
```
Comment (sayanh) [inline components/eventing-controller/README.md:57]: `WEBHOOK_ACTIVATION_TIMEOUT`, `EXEMPT_HANDSHAKE`, `QOS`, `CONTENT_MODE` are missing. You may find them [here](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/env/config.go#L36-L36).

**Review by sayanh — DISMISSED** [review]

File: components/eventing-controller/README.md [hunk 6]
```diff
@@ -30,31 +30,40 @@ make deploy-local-dry-run
...
+| **For BEB**                   |                                                                                     |
+| `TOKEN_ENDPOINT`              | The Authentication Server Endpoint to provide Access Tokens.                        |
+| `WEBHOOK_ACTIVATION_TIMEOUT`  | The timeout duration used for webhook activation to acquire Access Tokens for Kyma. |
+| `WEBHOOK_CLIENT_ID`           | The Client ID used by webhooks to acquire Access Tokens from Kyma.                  |
+| `WEBHOOK_CLIENT_SECRET`       | The Client Secret used by webhooks to acquire Access Tokens from Kyma.              |
+| `WEBHOOK_TOKEN_ENDPOINT`      | The Kyma public endpoint to provide Access Tokens.                                  |
+| `EXEMPT_HANDSHAKE`            | The exemption handshake switch of the subscription protocol settings.               |
+| `QOS`                         | The quality of service setting of the subscription protocol settings.               |
+| `CONTENT_MODE`                | The conten mode of the subscription protocol settings.                              |
```
Comment (alexandra-simeonova) [inline components/eventing-controller/README.md:65]:
```suggestion
| `CONTENT_MODE`                | The content mode of the subscription protocol settings.                              |
```

**Review by alexandra-simeonova — APPROVED** [review]

**Review by sayanh — APPROVED** [review]

**Pull request merged**